src/main/java/org/folio/rest/impl/VendorHelper.java

```diff
@@ -91,7 +92,12 @@ public VendorHelper(Map<String, String> okapiHeaders, Context ctx, String lang)
   public CompletableFuture<Errors> validateAccessProviders(CompositePurchaseOrder compPO) {
     CompletableFuture<Errors> future = new VertxCompletableFuture<>(ctx);
     Set<String> ids = verifyAndGetAccessProvidersIdsList(compPO);
```

Comment:
Looks like this set is no longer needed because this is the same as `poLinesMap.keySet()`